### PR TITLE
[11.0][UPD] dotfiles

### PR DESCRIPTION
**Pull request by thomaspaulb**
Fix CI

### CODE REVIEW

**Comment by thomaspaulb** [comment]
@OCA/web-maintainers Could you review?

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# web_widget_bokeh_chart
-bokeh==0.12.7
+# generated from manifests external_dependencies
+bokeh
```
Comment (StefanRijnhart) [inline requirements.txt]: Should the version be moved to https://github.com/OCA/web/blob/11.0/web_widget_bokeh_chart/__manifest__.py#L17?

Comment (thomaspaulb) [reply requirements.txt]: Since this is 11.0, i think it needs to be an `external_dependencies_override`, I've pushed it, lets see

**Review by legalsylvain — APPROVED** [review]
Thanks for removing travis !

File: requirements.txt (hunk 1 above)
Comment (StefanRijnhart) [reply requirements.txt]: Great, thanks!

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**